packages/elements-core/src/components/Docs/HttpOperation/Responses.tsx

```diff
@@ -43,14 +43,14 @@ export const Responses = ({ responses: unsortedResponses, onStatusCodeChange, on
           <TabList>
             {responses.map(({ code }) => (
               <Tab key={code} id={code}>
-                <Text color={code === activeResponseId ? 'primary' : 'muted'}>{code}</Text>
```

Comment:
This hasn't been necessary for a while, just cleaned it up as I went I hope that's ok.